awx/main/models/notifications.py

```diff
@@ -145,6 +183,14 @@ def display_notification_configuration(self):
 
 
 class Notification(CreatedModifiedModel):
+    def build_notification_succeeded_message(self):
+        # TODO: Get success template if available
```

Comment:
Same question re: these `TODO`s

Reply:
Did a search and the methods that I added here never get used. I think they were a part of an early draft that I forgot to remove. Dropping the methods.